### PR TITLE
Campaign incident owners section

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39611

## Description
Added a section to the Phishing Campaign incident which displays the incident owners and their quantity.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Review by idovandijk — COMMENTED** [review]
Section looks good and working correctly, as well as taking ownership.
I do think we can slightly improve the display with the help of @olgashain.
See if she can help with the HTML to make the padding consistent with the rest of the widgets as well as improve visibility of the owners themselves
image

If not possible, let's at least try to make it align to center when we expand the section so that we can use it expanded.

image

Olga what do you think?

**Comment by idovandijk** [comment]
Input from Olga:
1. we can reduce the distance between the number and the names
2. the section should be as narrow as others, it is ok if we make it tallerX2. Do no reduce the text font, it would too small to be legible and would loos the point why you added it
3. Don’t worry about consistency of the section heights. The consistency rule here shouldn’t apply, the section includes more information than just one number. Plus, we would reorganize it anyway. So we’ll find it a good spot for it.

File: Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py [hunk 1]
```diff
@@ -83,13 +83,37 @@ def perform_unlink_and_reopen(ids, action):
...
 
+def set_incident_owners(incident_ids, action, user_name):
+
+    incident_ids.append(demisto.incident()["id"])
+
+    for incident_id in incident_ids:
+        res = demisto.executeCommand("setIncident", {"id": incident_id, "owner": user_name})
+
+        if isError(res):
+            return_error(COMMAND_ERROR_MSG.format(action=action, ids=','.join(incident_ids)))
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py]: done

File: Packs/Campaign/Scripts/ShowCampaignIncidentsOwners/ShowCampaignIncidentsOwners.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Gets the campaign incident owners by their ids.
+
+    Args:
+        incident_ids: All the campaign incident ids.
+
+    Returns:
+        List of the incident owners.
+    """
+
+    incident_owners = set([demisto.executeCommand("getIncidents", {'id': incident_id})
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/Campaign/Scripts/ShowCampaignIncidentsOwners/ShowCampaignIncidentsOwners.py]: done.

**Comment by yaakovpraisler** [comment]
> @yaakovpraisler What happens if there are no owners to any of the incidents? Need to say "No incident owners." to be consistent with our other sections

Done.

**Review by dantavori — APPROVED** [review]

**Pull request merged**